lib/taskana-core/src/test/java/pro/taskana/impl/integration/TaskServiceImplIntAutocommitTest.java

```diff
@@ -359,6 +360,7 @@ public void shouldNotTransferByFailingSecurity() throws WorkbasketNotFoundExcept
         task.setId("");
         task.getWorkbasketSummaryImpl().setId(wbNoTransfer.getId());
         task.setWorkbasketKey(null);
+        task.setExternalId(IdGenerator.generateWithPrefix("TST"));
```

Comment:
Why are you using different prefix?

Reply:
Just to see that this works too. In the final product, only the external ids that were not set by the client (i.e. that are set implicitely by Taskana) will have the prefix "EID:". Currently the external ids supplied by the client don't have a prefix. There may be scenarios where it is beneficial to have a simple means to distinguish internally and externally supplied 'external_id's . In these cases, we could tag external_ids that really refer to an external id with a separate prefix ...